packages/main/src/plugin/experimental-feature-feedback-handler.ts

```
@@ -154,4 +180,83 @@ export class ExperimentalFeatureFeedbackHandler {
+    // Go through all experimental features (in this point we should have all properties set)
+    for (const [key, configuration] of this.experimentalFeatures) {
+      const featureGitHubLink = configurationProperties[key]?.experimental?.githubDiscussionLink;
+      // If the feature does not have a link or the dialog is disabled
+      if (!featureGitHubLink || configuration.disabled) continue;
+
+      // Compare timestamp of each experimental feature
+      const date = new Date();
+      if (configuration.remindAt && configuration.remindAt > date.getTime()) continue;
+      const featureName = formatName(key);
+
+      let footerMarkdownDescription: string = `:button[fa-thumbs-up]{command=openExternal args='["${featureGitHubLink}"]'} :button[fa-thumbs-down]{command=openExternal args='["${featureGitHubLink}"]'}`;
```

Biome (2.1.2)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)